frontend/src/metabase/visualizations/lib/settings/column.js

```diff
@@ -268,6 +268,7 @@ export const NUMBER_COLUMN_SETTINGS = {
         { name: t`Percent`, value: "percent" },
         { name: t`Scientific`, value: "scientific" },
         { name: t`Currency`, value: "currency" },
+        { name: t`Duration`, value: "duration" },
```

Comment:
We prefer not to backport new features to maintenance releases especially when they introduce new l10n strings because they will miss localization